### PR TITLE
#88 Using spec plugin, updating versions and update gitignore

### PR DESCRIPTION
**Pull request by arjantijms**
Signed-off-by: arjantijms <arjan.tijms@gmail.com>

### CODE REVIEW

File: impl/pom.xml [hunk 1]
```diff
@@ -440,18 +427,4 @@ Copyright &#169; 2019 Eclipse Foundation. All rights reserved.
...
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>findbugs-maven-plugin</artifactId>
-                <version>${findbugs.version}</version>
-                <configuration>
-                    <threshold>${findbugs.threshold}</threshold>
-                    <excludeFilterFile>${findbugs.exclude}</excludeFilterFile>
-                </configuration>
-            </plugin>
-        </plugins>
-    </reporting>
```
Comment (pnicolucci) [inline impl/pom.xml:456]: Why are we removing the findbugs plugin here?

Comment (arjantijms) [reply impl/pom.xml:456]: Less noise, smaller poms. IMHO finding bugs can much better be done by tools like sonar cloud.

**Comment by pnicolucci** [comment]
@arjantijms  I merged this but doing a pull on master and building I'm getting:

[ERROR] Failed to execute goal on project jakarta.servlet.jsp.jstl-api: Could not resolve dependencies for project jakarta.servlet.jsp.jstl:jakarta.servlet.jsp.jstl-api:jar:2.0.0-SNAPSHOT: The following artifacts could not be resolved: jakarta.el:jakarta.el-api:jar:4.0.0, jakarta.xml.bind:jakarta.xml.bind-api:jar:3.0.0: Failure to find jakarta.el:jakarta.el-api:jar:4.0.0 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
[ERROR]


thoughts?

**Comment by arjantijms** [comment]
@pnicolucci did you build with -Pstaging?

**Comment by pnicolucci** [comment]
@arjantijms , ah that does the trick, thanks for the tip! I'll cancel the PR to revert these changes and we should be back to normal once they are available out on maven central.

**Comment by arjantijms** [comment]
> ah that does the trick, thanks for the tip

You're welcome! :)

**Pull request merged**